docs: Fix build command & improve clarity (#47)
ReVanced/revanced-documentation: docs/revanced-development/1_setup.md

    
    ```bash
    for project in */; do

>         cd "$project" && ./gradlew build && cd ..

Reviewer: Why is .. necessary?

Line 30 should output full paths

Author: Not on my client, Git Bash for Windows. Here's the output of a test script:

```
for project in */; do
   echo "$project"
done
```
![image](https://github.com/user-attachments/assets/ad04406b-b597-49ef-8ba0-b653ab627694)

And so, only the first succeeds.

    done
    ```
    
> [!NOTE]
> If the build fails due to authentication, you may need to authenticate to GitHub Packages.

> > Create a Github Personal Access Token with the scope `read:packages` [here](https://github.com/settings/tokens/new?scopes=read:packages&description=ReVanced) and add your token to ~/.gradle/gradle.properties. If gradle.properties does not exist, you can create it.

Reviewer: ```suggestion
> Create a personal access tokens with the scope `read:packages` [here](https://github.com/settings/tokens/new?scopes=read:packages&description=ReVanced) and add your token to ~/.gradle/gradle.properties. Create the file if it does not exist.
```

>
> Example `gradle.properties` file:
>
> ```properties

> > gpr.user = <GitHub username>
> > gpr.key = <Generated Personal Access Token>

Reviewer: ```suggestion
> gpr.user = <GitHub username>
> gpr.key = <Personal access token>
```
